README.md

```diff
@@ -1,57 +1,11 @@
-# Northcoders House of Games API
+## How to use this repo
 
-## Background
+If you are using this repo, you will need to include a ".env.test" file, which points towards the test dataset using PGDATABASE = "Blah"
```

Comment:
These ideally need to be helpful instructions to set up the db. 
i.e. the database name. 

Imagine it is your first time on BE and these are the instructions, would you be able to successfully set up the .env files?

The text inside the .env is without spaces and quotation marks as well.